### PR TITLE
Add marcxml fetcher

### PR DESCRIPTION
**Pull request by calixtus**
ToDo:

* [ ] pages
* [ ] volume and number correct
* [ ] series and journals stored in different tags
* [ ] notes: 546-599
* [ ] thesis
* [ ] proceedings

- see https://www.loc.gov/marc/bibliographic/ for detailed documentation
- for modifications in B3Kat https://www.bib-bvb.de/documents/10792/9f51a033-5ca1-42e2-b2d3-a75e7f1512d4
https://www.dnb.de/DE/Professionell/Metadatendienste/Exportformate/MARC21/marc21_node.html
https://www.dnb.de/SharedDocs/Downloads/DE/Professionell/Standardisierung/AGV/marc21VereinbarungDatentauschTeil1.pdf?__blob=publicationFile&v=2
- multiple books in a series 
https://www.dnb.de/SharedDocs/Downloads/DE/Professionell/Standardisierung/marc21FormatumstiegAbbildungBegrenzterWerke2008.pdf?__blob=publicationFile&v=2

### CODE REVIEW

**Comment by koppor** [comment]
Ideas from DevCall:

Convert from marcxml to Dublin Core and then import.

grafik

Source https://www.loc.gov/standards/marcxml/

Other XSLT: https://gist.github.com/kardeiz/4504802

The drawback may be that German libraries do not follow that conversion.

Steps:

1. Check whether the XSLT covers the "German" codes. REsult: NO, For instance, 490 is not covered.
2. ~~If yes: Rewrite converter~~
3. If no: Continue here

**Comment by calixtus** [comment]
http://www.bib-bvb.de/web/b3kat/open-data

**Comment by calixtus** [comment]
Additional resources:
https://portal.dnb.de/opac.htm?method=simpleSearch&cqlMode=true&query=idn%3D1222522543
https://d-nb.info/1222522543/34

**Comment by calixtus** [comment]
Additional resources:
https://portal.dnb.de/opac.htm?method=simpleSearch&cqlMode=true&query=idn%3D1222522543
https://d-nb.info/1222522543/34
https://www.dnb.de/DE/Professionell/Metadatendienste/Exportformate/MARC21/marc21_node.html

File: src/main/java/org/jabref/logic/importer/fetcher/BvbFetcher.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return Optional.empty();
+    }
+
+    @Override
+    public URL getURLForQuery(QueryNode query) throws URISyntaxException, MalformedURLException {
+        URIBuilder uriBuilder = new URIBuilder(URL_PATTERN);
+        uriBuilder.addParameter("version", "1.1");
+        uriBuilder.addParameter("recordSchema", "marcxml");
+        uriBuilder.addParameter("operation", "searchRetrieve");
+        uriBuilder.addParameter("query", new DefaultQueryTransformer().transformLuceneQuery(query).orElse("")); // new BVBQueryTransformer().transformLuceneQuery(query).orElse(""));
```
Comment (koppor) [inline src/main/java/org/jabref/logic/importer/fetcher/BvbFetcher.java:37]: Remove comment?

**Comment by koppor** [comment]
Filed at https://github.com/JabRef/jabref/pull/9641

**Pull request closed without merging**